src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+//for generating random color
```

Comment:
Remove all comments, you're code must be readable without them's

Reply:
done